### ISSUE TITLE
[Accessibility Audit] Code Intel: Inspecting LSIF uploads for an instance (site-admin)

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#33514]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1kA6aVOAgID_uPm-d6uEC1DG6WHjJ8fPT0UAXQcML_KQ/edit?usp=sharing). Use this for further context.

- Navigate to site admin settings in Sourcegraph.
- Click on Uploads in the 🧠 Code Intelligence section in the sidebar.
- Rest of the steps are the same as for [Inspecting LSIF uploads for a repository](https://docs.google.com/document/d/1kA6aVOAgID_uPm-d6uEC1DG6WHjJ8fPT0UAXQcML_KQ/edit#heading=h.1t2ar7qztcrk)

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

**Comment by sourcegraph-bot-2** [comment]
Heads up @macraig - the "team/code-intelligence" label was applied to this issue.

**Comment by efritz** [comment]
Description of journey with screen brightness down:

- [x] Issues when navigating to target page will be the same as https://github.com/sourcegraph/sourcegraph/issues/33515
- [ ] Collapsible controls don't announce what happens
- [ ] Inconsistent pagination in filtered connections (without filter number at bottom; after filter number at top). This is the smoothest filtered connection experience so far (we were able to see how many were matching with relative ease).
- [ ] "No dependents" image should have some meaning: ![image](https://user-images.githubusercontent.com/103087/167895107-0c506f01-2495-4d08-835c-da7afc94cda2.png)
- [ ] "Search matches" for retention overview filter is initially confusing. Can have better placeholder text. 
![image](https://user-images.githubusercontent.com/103087/167895331-3d47942c-4772-4e33-ae7b-0b10d1cb6399.png)
- [ ] Image without meaning in retention policy description (I think the hoverable info icon). ![image](https://user-images.githubusercontent.com/103087/167895516-8f614781-7b02-4c7a-84b7-5290a9486877.png)

The remaining issues should be the same as https://github.com/sourcegraph/sourcegraph/issues/33512.
